src/features/game/types/expansions.test.ts

```diff
@@ -96,4 +96,41 @@ describe("getLand", () => {
     expect(land?.beehives).toHaveLength(1);
     expect(land?.flowerBeds).toHaveLength(1);
   });
+
+  it("returns spring resources if they had bought resource nodes", () => {
```

Comment:
Is this something only `spring` islands get to do?  Or something that could be done maybe every 3 months (like skill resets)?

Reply:
That's right 👍 

This is just an extra test to ensure that people who buy resources, don't discover less when expanding.